common/common.scss

```diff
@@ -27,6 +27,91 @@
   }
 }
 
-.scheme-icon {
-  margin-right: 0.25em;
+// theme toggle button
+
+/* The switch - the box around the slider */
+#dark-light-toggle {
+  display: flex;
+  width: auto;
+
+  &:hover {
+    background-color: transparent;
+    .selected .d-icon {
+      color: transparent;
+    }
+  }
+}
+
+.switch {
+  position: relative;
+  width: 70px;
+  height: 35px;
+  margin-bottom: 0;
+  display: flex;
+  align-items: center;
+  border-radius: 35px;
+  border: 1px solid var(--primary-low-mid);
+  justify-content: space-between;
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.switch span.toggle-icon {
+  display: flex;
+  align-items: center;
+  margin: 0 0.5em;
+  .d-icon {
+    width: 20px;
+    color: var(--primary-medium);
+    font-size: 20px;
+    align-self: center;
+  }
+}
+
+/* The slider */
+.slider {
+  position: absolute;
+  cursor: pointer;
+  height: 18px;
+  width: 18px;
+  top: calc(50% - 12px);
+  left: 4px;
+  right: 0;
+  bottom: 0;
+  border-radius: 50%;
+  border: 3px solid var(--primary-low-mid);
+  -webkit-transition: transform 0.2s;
+  transition: transform 0.2s;
+  background-color: var(--primary-low);
+}
+
+.switch.light {
+  .slider {
+    border-color: var(--secondary-high);
+    background-color: var(--secondary-very-high);
+  }
+
+  span.toggle-icon .d-icon-sun {
+    color: transparent;
+  }
+}
+
+.switch.dark {
+  span.toggle-icon .d-icon-sun {
+    color: var(--primary-medium);
+  }
+  span.toggle-icon .d-icon-far-moon {
+    color: transparent;
+  }
+}
+
+span:focus + .slider {
+  box-shadow: 0 0 1px #2196f3;
+}
+
+.switch.dark span.slider {
+  -webkit-transform: translateX(36px);
+  -ms-transform: translateX(36px);
```

Comment:
1. Similarly to `transition`, webkit uses nonprefixed property since 2015
2. We no longer support IE and on edge the property isn't prefixed.